src/onegov/org/elements.py

```diff
@@ -276,3 +276,50 @@ def __init__(
     'LinkGroup',
     'QrCodeLink'
 )
+
+
+class iFrameLink(BaseLink):
```

Comment:
```suggestion
class IFrameLink(BaseLink):
```
Classes should always start with an uppercase letter, this is also more consistent with `IFrameForm`.